pkg/util/overridemanager/overridemanager.go

```diff
@@ -26,6 +26,16 @@ type OverrideManager interface {
 	ApplyOverridePolicies(rawObj *unstructured.Unstructured, cluster string) (appliedClusterPolicies *AppliedOverrides, appliedNamespacedPolicies *AppliedOverrides, err error)
 }
 
+// OverridePolicyObject is an abstract object of ClusterOverridePolicy and OverridePolicy
+type OverridePolicyObject interface {
```

Comment:
```suggestion
type GeneralOverridePolicy interface {
```
To eliminate the confusion between the interface name and API name.

Reply:
Updated and rebased.